src/osx/cocoa/textctrl.mm

```diff
@@ -773,6 +773,12 @@ - (void)controlTextDidEndEditing:(NSNotification *)aNotification
 
     [tv setDelegate: tv];
 
+    m_undoManager = nil;
+    if ( wxPeer && wxPeer->GetParent() && wxPeer->GetParent()->GetPeer() && wxPeer->GetParent()->GetPeer()->GetWXWidget() )
```

Comment:
@Dan-Korn I wonder if we want to use the parent here or the TLW parent? I.e. shouldn't this use `wxGetTopLevelParent()`?